src/Datasource/QueryTrait.php

```diff
@@ -388,7 +388,7 @@ public function getMapReducers(): array
      * @return $this
      * @throws \InvalidArgumentException
      */
-    public function formatResults(?callable $formatter = null, $mode = 0)
+    public function formatResults(?callable $formatter = null, $mode = self::APPEND)
```

Comment:
this defined in `ORM/Query` not here in trait

Reply:
You might have to make a follow up PR here.